docs/tasks/tools/install-minikube.md

```diff
@@ -29,6 +29,9 @@ If you do not already have a hypervisor installed, install one now.
 [VirtualBox](https://www.virtualbox.org/wiki/Downloads) or
 [KVM](http://www.linux-kvm.org/).
 
+   **Note:** Minikube also supports a `--vm-driver=none` option that runs the Kubernetes components on the host and not in a VM.  Docker is required to use this driver but no hypervisor.
```

Comment:
Change the last sentence:
> Docker is required to use this driver but a hypervisor is not required.

@Rajakavitha1 From #7394, do you mind also changing the note in https://github.com/kubernetes/minikube#requirements to use the same language? ☝️ It's OK if that change is in this PR or a separate one. 🙇